mappings/net/minecraft/client/options/GameOption.mapping

```diff
@@ -3,6 +3,7 @@ CLASS cve net/minecraft/client/options/GameOption
 	FIELD B CHAT_COLOR Lcup;
 	FIELD C CHAT_LINKS Lcup;
 	FIELD D CHAT_LINKS_PROMPT Lcup;
+	FIELD E DISCRETE_MOUSE_SCROLL Lcup;
```

Comment:
Overlap #610, and the ingame name is "discrete scrolling"

Reply:
Ah, I missed that